src/query/v2/shard_request_manager.hpp

```diff
@@ -227,7 +228,7 @@ class ShardRequestManager : public ShardRequestManagerInterface {
     return str;
   }
   const std::string &LabelToName(memgraph::storage::v3::LabelId /*label*/) const override {
-    static std::string str{"dummy__label"};
+    static std::string str{"label"};
```

Comment:
You can use `shards_map_.GetLabelName` to return a proper value. Otherwise this function won't work for labels apart for "label". That would just push the hard coded value into a different place, but won't really help us with other labels.